h5c/vic/src/vic-webapp/src/app/container-view/container-view.template.html

```diff
@@ -6,18 +6,29 @@
           <clr-alert-item *ngIf="errorObj.type === 'ssl_cert'">
             <span class="alert-text">
               <!-- TODO: i18n -->
-              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Your browser may be blocking the connection because of the untrusted SSL certificate.
+              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Please check the following:<br/>
+              1. VIC appliance VM is up, but unreachable due to network misconfigurations<br/>
+              2. VIC appliance VM is up and reachable, but API endpoint is unreachable due to possible issues that include, but not limited to:<br/>
+              &nbsp;&nbsp;- Network misconfiguration<br/>
+              &nbsp;&nbsp;- Firewall rules<br/>
+              &nbsp;&nbsp;- Endpoint being down<br/>
```

Comment:
Can we format this as a `ul` within an `ol` instead of using breaks and non-breaking spaces?